### DISCUSSION TITLE
ModernRIBs 디펜던시 에러

### DISCUSSION THREAD

**Discussion by Jinsujin** [nsoojin/MiniSuperApp-fastcampus#3] [아무 질문이나 환영합니다] [answered]
에러는 아래와 같이 뜹니다:
```
because ModernRIBs >=0.0.2 contains incompatible tools version and root depends on ModernRIBs 1.0.1..<2.0.0, version solving failed.
```
안녕하세요, 디펜던시 에러가 해결이 안되서 질문드리러 왔습니다.
실행 환경문제인것 같은데,  현재 제가 사용하고 있는  Xcode 버전은 12.4 입니다.    
**ModernRIBs: Xcode 11.0 or later** 로 명시가 되어있는데,  더 확인할 사항이 있을까요?

**Comment by nsoojin** [comment, marked as answer]
제가 직접 해본바로는 12.5에서도 잘 됐었는데 12.4는 안해봐서 모르겠습니다. 
Package.swift를 쓰고 계신가요? 소스파일 맨 위에 `// swift-tools-version:5.3` 버전이 12.4랑 맞는 버전인지 확인 해주세요.

**Reply by nsoojin** [reply]
ModernRIBs 패키지가 `// swift-tools-version:5.4` 로 되어있는데([파일](https://github.com/DevYeom/ModernRIBs/blob/main/Package.swift)) 5.4는 Xcode 12.5부터 지원인거 같습니다. 엑스코드를 업데이트하시거나, ModernRIBs가 `// swift-tools-version:5.3` 이하로 수정되어야 Xcode 12.4에서도 쓸 수 있을거 같습니다.

**Reply by Jinsujin** [reply]
확인해 주셔서 정말 감사합니다 🙏🏼
12.5 설치후, 잘 되는것을 확인했습니다
